.github/ISSUE_TEMPLATE/community-call-anonymised-discussion.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Community Call Anonymised Discussion
+about: Use this to create an issue for anonymising the community call discussion document
+  as per Chatham House rule
```

Comment:
```suggestion
about: Use this to create an issue for anonymising the community call discussion document as per Chatham House rule
```

Reply:
I don't know if the template will work if this text is broken up over multiple lines. Hence the proposed fix.